Catalog/src/Application/Common/Security/AuthorizeAttribute.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Catalog.Application.Common.Security;
+
+/// <summary>
+/// Specifies the class this attribute is applied to requires authorization.
+/// </summary>
+[AttributeUsage(AttributeTargets.Class, AllowMultiple = true, Inherited = true)]
+public class AuthorizeAttribute : Attribute
```

Comment:
Isn't this the same as `default` ASP.net core `Authorize` attribute?
Knowing DDD and patterns doesn't mean you need to follow them blindly :)

Reply:
Oh, that's another leftover. I've removed it for now, will have a look at when it's time for the authentication/authorization module.